### ISSUE TITLE
MAL Unstable login false api url.

### ISSUE THREAD

**Issue by DaniGTA** [PolyMeilex/node-myanimelist#15]
> Installed version: node-myanimelist: 3.0.3

On:
```js
const malAuth = Mal.auth();
const acount = await malAuth.Unstable.login(username, pass);
```
It makes a post request to this url: ```https://api.myanimelist.net/v2//auth/token```
but it must be: ```https://api.myanimelist.net/v2/auth/token```

**Comment by PolyMeilex** [comment]
Thanks, I'll fix that asap.

**Comment by PolyMeilex** [comment]
Sorry those where leftovers from previous architecture, I should have unit test for those, my bad

**Comment by PolyMeilex** [comment]
Fixed in version [v3.0.4](https://github.com/PolyMeilex/node-myanimelist/commit/2907b5fa94a742ee676602e74ae9674032f7302c)
